src/models/Transform.js

```diff
@@ -23,17 +23,37 @@ const transformOptions = {
     title: 'Grow'
   },
   growValue: {
-    title: 'Scale step (+/-)',
+    title: 'Scale (+/-)',
+  },
+  growAdvanced: {
+    title: 'Advanced',
+    type: 'checkbox',
+  },
+  growMath: {
+    title: 'Scale Function (i)',
+    type: 'text',
+    isVisible: state => { return state.growAdvanced },
   },
   spinEnabled: {
-    title: 'Spin'
+    title: 'Spin',
+    isVisible: state => { return !state.spinAdvanced },
   },
   spinValue: {
-    title: 'Spin step (+/-)',
-    step: 0.1
+    title: 'Spin (+/-)',
+    step: 0.1,
+  },
+  spinAdvanced: {
+    title: 'Advanced',
+    type: 'checkbox',
+  },
+  spinMath: {
+    title: 'Spin Function (i)',
```

Comment:
Reminder that, whatever names we settle on, that first word only is capitalized.